.gitignore

```
@@ -165,6 +165,7 @@ mom6/
 oops/
 saber/
 saber-data/
+soca/
```

I think @guillaumevernieres explicitly removed this previously because then it ignores directories like `parm/config/soca` 

```suggestion
/soca/
```

---

As a test I modified `parm/soca/obs/obs_list.yaml`.  Whether `.gitignore`  has `soca/` or `/soca/`, "git status" returns `parm/soca/obs/obs_list.yaml` as modified.   I made this test on Hera with git version 2.18.0.   Shall we retain `soca/`?

---

@RussTreadon-NOAA , `git status` won't show new files under `soca` if soca is in `.gitignore`. We could rename that directory in the future I suppose.

---

According to google/stackoverflow, `/soca/` should do what we want it to do, only ignore the `soca` directory that is cloned by ecbuild. But I haven't tested this myself.

---

> According to google/stackoverflow, `/soca/` should do what we want it to do, only ignore the `soca` directory that is cloned by ecbuild. But I haven't tested this myself.

Yup, just tested, `/soca/` does what we need! Thanks @CoryMartin-NOAA for pointing it out.

---

[8f90235](https://github.com/NOAA-EMC/GDASApp/commit/8f902357365351f122d91ec13948c4e455012df2) replaces `soca/` with `/soca/` in `.gitignore`.   Do we want this change for any other entries in `.gitignore`?

---

For now I think this is fine, `soca` is the only one I can think of that we will have a subdirectory for. Perhaps `ufo`, `ioda` or `fv3`? But we can cross that bridge when we get to it I think